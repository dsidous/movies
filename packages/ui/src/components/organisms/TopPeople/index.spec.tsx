import React from 'react';
import { shallow } from 'enzyme';
import TopPeopleProfile from '.';

const mockProps = {
  toppeople: [
    {
      id: 1,
      name: 'name',
      profile_path: 'path/to/image',
    },
  ],
  page: 1,
  handlePageSelect: jest.fn(),
};

describe('Organisms/TopPeopleProfile', () => {
  it('should render as expected', () => {
    const wrapper = shallow(<TopPeopleProfile {...mockProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
