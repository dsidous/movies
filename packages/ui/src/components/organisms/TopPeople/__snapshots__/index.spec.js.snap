// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organisms/TopPeopleProfile should render as expected 1`] = `ShallowWrapper {}`;
